api/src/test/java/org/openmrs/module/fhirExtension/export/anonymise/handler/AnonymiseHandlerTest.java

```diff
@@ -93,6 +96,60 @@ public void shouldAnonymisePatientResource_WhenPatientResourceAndPatientResource
 		assertEquals(Calendar.JANUARY, patient.getDeceasedDateTimeType().getMonth());
 	}
 	
+	@Test
+	public void shouldAnonymisePatientResourceWithRandomise_WhenPatientResourceAndPatientResourceTypePassed() {
```

Comment:
Keep method name relevant to json configuration i.e. shouldAnonymisePatientResourceWithRandomValue_WhenPatientResourceTypeFieldsAreConfiguredWithMethodAsRandom